src/types/redis_json.cc

```diff
@@ -102,14 +108,18 @@ rocksdb::Status Json::Get(const std::string &user_key, const std::vector<std::st
   if (paths.empty()) {
     res = std::move(json_val);
   } else if (paths.size() == 1) {
-    auto get_res = json_val.Get(paths[0]);
+    auto json_path_result = JsonPath::BuildJsonPath(paths[0]);
+    if (!json_path_result) return rocksdb::Status::InvalidArgument(json_path_result.Msg());
+    auto get_res = json_val.Get(json_path_result.GetValue());
```

Comment:
I think maybe there's no difference between the original code and this patch here.